src/pages/ResearcherProfile.js

```diff
@@ -379,42 +397,52 @@ export default function ResearcherProfile(props) {
                 style: { 'marginTop': '20px' },
                 isRendered: hasInstitution() && !isSigningOfficial(),
               }, [
-                label({
-                  id: 'lbl_profileInstitution',
-                  className: 'control-label',
-                  style: headerStyle,
-                }, [
-                  'Signing Official* ',
-                ]),
-                p({}, ['Please select your Signing Official or enter your signing official’s email address. This is an individual who has the authority to engage your institution in legal contracts, likely someone in your grants department or office of sponsored research.']),
-                div({},
-                  [
-                    h(SearchSelectOrText, {
-                      id: 'SigningOfficial',
-                      label: 'SigningOfficial',
-                      onPresetSelection: (selection) => {
-                        setProfile(Object.assign({}, profile, {selectedSigningOfficialId: selection, suggestedSigningOfficial: undefined}));
-                      },
-                      onManualSelection: (selection) => {
-                        if (selection == '') {
-                          setProfile(Object.assign({}, profile, {selectedSigningOfficialId: undefined, suggestedSigningOfficial: undefined}));
-                        } else {
-                          setProfile(Object.assign({}, profile, {selectedSigningOfficialId: null, suggestedSigningOfficial: selection}));
-                        }
-                      },
-                      options: signingOfficialList.map(signingOfficial => {
-                        return {
-                          key: signingOfficial.userId,
-                          displayText: signingOfficial.displayName,
-                        };
-                      }),
-                      placeholder: 'Please Select a Signing Official',
-                      searchPlaceholder: 'Search for a Signing Official...',
-                      value: profile.selectedSigningOfficialId,
-                      freetextValue: profile.suggestedSigningOfficial,
-                      className: 'form-control'
-                    })
-                  ]),
+                h(FormField, {
```

Comment:
The form field error for `suggestedSigningOfficial` shows up if a user selects a SO from their institution. You can replicate this by selecting an SO from the list, save the changes to your profile, and then either revisit or reload the Researcher Profile Page.

<img width="1578" alt="Screen Shot 2022-10-05 at 8 32 47 AM" src="https://user-images.githubusercontent.com/6664778/194063325-242c7e09-a806-496a-a798-2d13940418c1.png">

Reply:
Wasn't quite able to recreate this, but I should have fixed it by only displaying the error if there is no institution id.

Reply:
The validation error is still there. Are you checking to see if `currentSigningOfficial` is populated?

![Kapture 2022-10-06 at 13 59 48](https://user-images.githubusercontent.com/6664778/194386553-180fba6f-8cd0-42cc-8840-cafdb83c4a00.gif)

Reply:
Hmm curious, theoretically they should be the same but I added that check too.